glean-core/ios/Glean/Generated/GleanBaseline.swift

```diff
@@ -0,0 +1,29 @@
+// -*- mode: Swift -*-
+
+// AUTOGENERATED BY glean_parser.  DO NOT EDIT.
+// MANUALLY UPDATED. WILL BE AUTO-GENERATED IN THE FUTURE.
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+internal enum GleanBaseline {
```

Comment:
internal is redundant here as it is the default

Reply:
Oh true. I actually took the output of the glean_parser, so I guess ... I need to fix t hat.

Reply:
Don't block on this for this, I know it's a glean_parser thing.